File: Chapter 02/C Programs/2_13.c
#include<stdio.h>
void main()
{
int a, b,c;
printf("Enter 3 numbers: ");
scanf("%d%d%d", &a, &b, &c);
if (a>b)
{
	if (b>c)
		printf( "%d is largest", a);
	else if (a>c)
		printf( "%d is largest", a);
	else
		printf( "%d is largest", c);
}
else
{
	if (a>c)
		printf( "%d is largest", b);

	else if ( b>c)
		printf( "%d is largest", b);
    else
		printf( "%d is largest", c);
}
}

File: Chapter 02/Python programs/2_13.py
a=int(input("Enter first number"))
b=int(input("Enter first number"))
c=int(input("Enter first number")) 
if a>b:
    if b>c:
        print("a is largest")
    elif a>c:
        print("a is largest")
    else:
        print("c is largest")
else:
    if a>c:
        print("b is largest")
    elif b>c:
        print("b is largest")
    else:
        print("c is largest")
